lighthouse-core/audits/dobetterweb/no-mutation-events.js

```diff
@@ -61,13 +61,7 @@ class NoMutationEventsAudit extends Audit {
    * @return {!AuditResult}
    */
   static audit(artifacts) {
-    if (typeof artifacts.EventListeners === 'undefined' ||
-        artifacts.EventListeners === -1) {
```

Comment:
right. I think this may have been from an earlier state where it was using `this.artifact = -1` as the error signal, it was changed when we wanted a `debugString` to come through for errors, but this conditional was never updated.